src/main/java/org/traccar/protocol/MictrackProtocolDecoder.java

```diff
@@ -142,7 +162,15 @@ private void decodeStatus(Position position, String data) throws ParseException
     @Override
     protected Object decode(
             Channel channel, SocketAddress remoteAddress, Object msg) throws Exception {
+        if (((String) msg).startsWith("MT")) {
```

Comment:
One last thing. Can you please downcast to `String` once and then just pass string around as a `sentence` variable.

Reply:
Done.